pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go

```diff
@@ -418,3 +418,23 @@ func (qtw *RuntimeQuotaCalculator) logQuotaInfoNoLock(verb string, quotaInfo *Qu
 		quotaInfo.CalculateInfo.Runtime, quotaInfo.CalculateInfo.Used, qtw.treeName, qtw.totalResource, qtw.groupReqLimit,
 		qtw.globalRuntimeVersion)
 }
+
+func getValue(res resource.Quantity, resName v1.ResourceName) int64 {
```

Comment:
make the name more meaningful. maybe getQuantityValue is better?

Reply:
fixed